src/visitors/MethodAndRemoteMethodCountVisitor.java

```diff
@@ -0,0 +1,40 @@
+package visitors;
+
+import com.github.javaparser.ast.Node;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.body.MethodDeclaration;
+import com.github.javaparser.ast.expr.MethodCallExpr;
+import com.github.javaparser.ast.visitor.GenericVisitorAdapter;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class MethodAndRemoteMethodCountVisitor extends GenericVisitorAdapter<Integer, Void> {
+
+    @Override
+    public Integer visit(ClassOrInterfaceDeclaration n, Void arg) {
+        // returns number of methods and methodCallExprs within those methods
+        return n.getMethods().size() + super.visit(n, arg);
+    }
+
+    @Override
+    public Integer visit(MethodDeclaration n, Void arg) {
```

Comment:
This method is not getting called past the first method in the analyzed file.

Reply:
This is happening coz of `GenericVisitorAdapter`

> A visitor that has a return value (R), and has a default implementation for all its visitmethods that visits their children in an unspecified order, and the first visit methodthat returns a value will stop the visitation and be the end result.

So maybe store the value as an instance variable and add to it. Even better if you put all the remote calls in one instance set.